conf/timeseries-filodb-server.conf

```diff
@@ -5,13 +5,19 @@ filodb {
     port = 9042
     partition-list-num-groups = 1
   }
-  spread {
-    default = 1
-    override {
-      App-0 = 2
-      App-5 = 0
+  spread-default = 1
+
+  # Override default spread for application using override block which will have non metric shard keys and spread
+  spread-override = [
+    {
+      _ns = App-0,
+      spread = 2
```

Comment:
Just one thought.  The design of config means that there cannot be a shard label named "spread", correct?  This should be documented.

Reply:
Yes, I have added it in conf/timeseries-filodb-server.conf

Reply:
That's a good idea. Cjanged it to _spread_